apps/financial-aid/web-shared/tsconfig.app.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Not sure if we want to got this router, but added it as a suggestion, if we want to go this route then I'll add a alias for the import in another PR.